backend/src/routes/userRoute.ts

```diff
@@ -0,0 +1,66 @@
+import express, { Request, Response } from "express";
+import admin, { ServiceAccount } from "firebase-admin";
+
+import * as serviceAccount from "../firebase/ServiceAccountKey.json";
+import User, { UserDocument } from "../models/User";
+
+// Define the type for req.body
+type CreateUserRequestBody = {
+  name: string;
+  gender: string;
+  accountType: "admin" | "team";
+  approvalStatus: string; // NOTE Should this be restricted to certain values?
+  email: string;
+  password: string;
+};
+
+const router = express.Router();
+
+// Initialize Firebase Admin SDK
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount as ServiceAccount), // Type assertion
+  databaseURL: process.env.FIREBASE_DATABASE_URL,
+});
+
+router.use(express.json());
+
+router.post(
```

Comment:
We need some validator function to ensure that the req body is correct. See [this](https://github.com/TritonSE/onboarding/tree/main/backend/src/validators) for an example.